iptables/pkg.yaml

```diff
@@ -15,18 +15,20 @@ steps:
       - |
         tar -xf iptables.tar.xz --strip-components=1
 
+        patch -p1 < /pkg/patches/0001-fix-redefinition-on-musl.patch
```

Comment:
Weird how it broke. May be somehow related to relocation of headers

Reply:
just add a comment on top of patch from where it was obtained from, usually alpine has required patches

Reply:
Did it myself just removing those duplicate includes